apex/contrib/examples/nccl_allocator/change_cuda_allocator.py

```diff
@@ -0,0 +1,14 @@
+import torch
+import apex.contrib.nccl_allocator as nccl_allocator
+
+nccl_allocator.init()
+nrep = 6
+with nccl_allocator.nccl_mem():
+    for i in range(nrep):
+      out = torch.randn(1024).cuda()
+
+for i in range(nrep):
+  out = torch.randn(1024).cuda()
+
+torch.cuda.empty_cache()
+torch.cuda.empty_cache()
```

Comment:
[question] why calling twice?

Reply:
It is just for a sanity check. I had a bug that causes it to segfault if `empty_cache` was called twice.